src/extensions/default/bramble/stylesheets/consoleTheme.less

```diff
@@ -0,0 +1,279 @@
+#editor-console {
+    position: relative;
+}
+
+#editor-console .console {
+    overflow-x: hidden;
+    max-height: inherit;
+    height: calc(100% - 44px) !important;
```

Comment:
@humphd Need to escape the calc rule here...

``height: ~"calc(100% - 44px)" !important;``